addons/hr_timesheet/views/hr_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="hr_employee_view_form_inherit_timesheet" model="ir.ui.view">
+        <field name="name">hr.employee.form.timesheet</field>
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='active_group']" position="before">
+                <!-- Empty invisible group to be filled by timesheet_grid and sale_timesheet -->
```

Comment:
Why make it invisible? Are there cases where hr_timesheet will be installed explicitly but you won't have anything else?